Assets/Scripts/Requests/Requests.cs

```diff
@@ -41,6 +41,7 @@ private void Awake()
 
         public void Add(Request request)
         {
+            Debug.Log(request.name);
```

Comment:
Any of these log statements you're keeping should probably be labelled. Also, having a way to turn on only some of them would be nice.